python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/radius_servers.schema.yml

```diff
@@ -9,6 +9,7 @@ keys:
   radius_servers:
     deprecation:
       warning: true
+      removed: true
       new_key: radius_server.hosts
       remove_in_version: v5.0.0
     type: list
```

Comment:
Please remove `items` and everything below, since it only fills up the table.

Reply:
Removed.